lib/rom/support/inflector.rb

```diff
@@ -13,8 +13,11 @@ module Inflector
           ::Inflecto
         end
     rescue LoadError
-      raise 'Unable to find an inflector library'
+      raise LoadError,
+            "No inflector library could be found: "\
+            "please install either the `activesupport` or `inflecto` gem."
```

Comment:
Not really actually, if you're using Rails we want AS, you could have custom inflections configured etc.